Profile/ProfileTests/Presentation/DeleteAccount/DeleteAccountViewModelTests.swift

```diff
@@ -9,6 +9,7 @@ import SwiftyMocky
 import XCTest
 @testable import Core
 @testable import Profile
+import OEXFoundation
 import Alamofire
```

Comment:
If you moved Alamofire to OEXFoundation i think then we should remove import that framework from main base code. Need to implement accessors for alamofire requests. For my side it's will be logic that you just import OEXFoundation and able to use Alamofire's methods. Accessors could be named same as Alamofire's methods.

Same for other moved frameworks.

Reply:
Thank you for the suggestion! We discussed the idea of creating wrappers for all libraries, including Alamofire, and concluded that at this stage, it would add complexity and slow down development.
The main reasons we decided to stick with direct imports for now are:

**Need to cover all functionalities** — creating wrappers for all required methods would take significant time, and in the future, it could slow down development as each new need would require adding a wrapper first.

**Potential loss of flexibility** — using libraries directly provides immediate access to all their functionalities. With wrappers, access to new or specialized functions may be limited, making development more challenging.

**Increased maintenance costs** — every new library version might require updating wrappers and verifying compatibility, leading to additional maintenance overhead.

We’ll definitely revisit this idea if a more isolated architecture is needed, but for now, we’ve decided to keep direct imports to ensure greater flexibility and development speed.